memory_ttl_test.go

```diff
@@ -29,6 +29,32 @@ func TestMemoryCacheTTL(t *testing.T) {
 	}
 }
 
+func TestMemoryCacheTTLGetExpired(t *testing.T) {
+	// Needs go test -race to catch problems
```

Comment:
All I meant was that this test won't catch any problems unless -race is set - if you have that then great!